Formula/vulkan-tools.rb

```diff
@@ -0,0 +1,106 @@
+class VulkanTools < Formula
+  include Language::Python::Virtualenv
+
+  desc "Vulkan Utilities and Tools"
+  homepage "https://github.com/KhronosGroup/Vulkan-Tools"
+  url "https://github.com/KhronosGroup/Vulkan-Tools/archive/refs/tags/v1.3.247.tar.gz"
+  sha256 "3c7f9d81d599a36871cc122501713cd67882be4777872c8c5e17834e2c2047a1"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-Tools.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "hatch" => :build
+  depends_on "python@3.11" => :build
+  depends_on "glslang"
+  depends_on "vulkan-headers"
+  depends_on "vulkan-loader"
+
+  on_macos do
+    depends_on "molten-vk"
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "libx11"
+    depends_on "libxcb"
+    depends_on "libxkbfile"
+    depends_on "libxrandr"
+    depends_on "wayland"
+    depends_on "wayland-protocols"
+  end
+
+  resource "jsonschema" do
+    url "https://files.pythonhosted.org/packages/36/3d/ca032d5ac064dff543aa13c984737795ac81abc9fb130cd2fcff17cfabc7/jsonschema-4.17.3.tar.gz"
+    sha256 "0f864437ab8b6076ba6707453ef8f98a6a0d512a80e93f8abdb676f737ecb60d"
+  end
+
+  def install
+    venv = virtualenv_create(buildpath/"venv", "python3")
+    resource("jsonschema").stage do
+      system Formula["hatch"].bin/"hatch", "build", "-t", "wheel"
+      venv.pip_install Dir["dist/*.whl"].first
+    end
```

Comment:
Are you sure this is used? It doesn't look like you're using it anywhere.

Reply:
It appears not!  This was put in because it was a stated requirement in the build docs of the source repo, but if it's not actually needed I'm happy to have it gone!